ds3-sdk/src/test/java/com/spectralogic/ds3client/Ds3ClientBuilder_Test.java

```diff
@@ -31,6 +32,27 @@ public void createBasicClient() throws Exception {
     }
 
     @Test
+    public void createBasicClientFromENV() throws Exception {
+        try {
+            final Ds3ClientBuilder builder = Ds3ClientBuilder.fromEnv();
```

Comment:
this is a good test, but you need first to set the environment variables before trying to build from env. 
and use remove the "" from the getenv argument (same comment as before)